### DISCUSSION TITLE
Publiczny dostęp do Allegro Ads Api

### DISCUSSION THREAD

**Discussion by jakubmaguza** [allegro/allegro-api#4487] [Suggestion - verification]
Tak jak w wątku https://github.com/allegro/allegro-api/issues/3152
Może tutaj uda się nazbierać więcej głosów i udostępnicie to publicznie szybciej.

**Comment by javaguru-pl** [comment]
Z tego co pamiętam w jednej z ostatnich ankiet REST API było pytanie o to "jakie funkcjonalności chciałbyś wdrożyć w API?". Wśród odpowiedzi była opcja API dot. ADS-ów. Czy można zatem prosić o oficjalne stanowisko Allegro w tej sprawie? Czy jest to w planach, a jeżeli tak to kiedy?

**Reply by AureliuszBrussy** [reply]
Aktualnie nie jestem w stanie podać ostatecznego stanowiska. Jeżeli zdecydujemy się na upublicznienie API do obsługi Allegro Ads poinformujemy Was o tym w tym wątku.

**Reply by bswierzewski** [reply]
Wiadomo coś więcej w tej sprawie? Przyznam szczerze że dostęp do statystyk w allegro ads byłby bezcenny.

**Reply by AureliuszBrussy** [reply]
Na ten moment nie upubliczniamy zasobów Allegro Ads. Jeżeli coś się zmieni w tej sprawie - będziemy to komunikować.

**Comment by tomasz-grzesik** [comment]
Proszę o ponowne rozważenie udostępnienia dostępu do API Allegro Ads wszystkim developerom.
Aktualnie nie wygląda to za dobrze: dostęp dla wybranych, Allegro Ads Partner, czy obchodzenie API.

**Reply by javaguru-pl** [reply]
Dodam tylko, iż ograniczenie to dotyczy dostępu do WŁASNYCH danych. Dane dot. kosztów ADS to przecież dane dotyczące usługi reklamowej wykonywanej przez Allegro na rzecz sprzedawcy. Tym samym CELOWE ograniczanie dostępu przez API do tych danych (gdyż API już od dłuższego czasu działa - tylko nie jest publicznie dostępne) - jest EWIDENTNYM łamaniem ustawy o ochronie konkurencji i konsumentów Art. 9. (i to conajmniej w kilku podpunktach ust. 2 tego art.).

**Reply by cgchow94** [reply]
@javaguru-pl @AureliuszBrussy  hey, do I understand it correctly, that currently I (an Allegro seller), am not able to get the ad cost and performance per offer on a daily basis of my **own offers via API?**

I see the classified end point, that is however not providing financial performance related ad data for me.

E.g. also within the billing data, I cannot see any granularity on offer level.
<img width="889" height="65" alt="Screenshot 2026-01-02 at 12 09 13" src="https://github.com/user-attachments/assets/f8a3f029-98b0-4cc8-b0f3-ffa514e33046" />

  {
    "id": "caee9a38-8307-5750-b287-53f76c435160",
    "occurredAt": "2025-11-28T03:12:36.031Z",
    "type": {
      "id": "NSP",
      "name": "Opłata za kampanię Ads"
    },
    "value": {
      "amount": "-10.20",
      "currency": "PLN"
    },
    "tax": {
      "percentage": "23.00"
    },
    "balance": {
      "amount": "-43.63",
      "currency": "PLN"
    },
    "additionalInfo": [
      {
        "type": "feeDate",
        "name": "Opłata za dzień",
        "value": "25.11.2025"
      }
    ]
  },

**Reply by Lukasz-Zurek** [reply]
@javaguru-pl Z uwagi na czynniki biznesowe, Allegro nie zdecydowało się dotychczas na publiczne udostępnienie API dla Allegro ADS. Podkreślę tu jednak, że jako zespół świadczący wsparcie API na tym forum zawsze staramy się przekłuć potrzebę w realny rozwój i realizację. Przekazujemy każdą tego typu sugestię (a było ich dotychczas sporo), a zarazem potrzebę, do odpowiednich zespołów. Nie mamy tutaj jednak pełnego wpływu na dalsze decyzje. Proszę o wyrozumiałość.

@cgchow94 I can confirm it. Ads support is not available within the Allegro public API. If we decide to make the Allegro Ads API publicly available, we'll let you know in this thread.

**Comment by zvapo** [comment]
Hej, 

Także podpisuje się pod petycją o udostępnienie API do Allegro ADS :)

Chodzi od dostęp do danych i możliwość zarządzania kampaniami reklamowymi. 
Przy wystawianiu wielu ofert, organizacja ADS nawet za pomocą aplikacji Allegro Ads Partner jest bolesne i zbyt czasochłonne. Przez to w większości przypadków ADS żyją swoim życiem i nikt nie chce tam zaglądać.

**Reply by Toumash** [reply]
@Zvapo dla takiego przypadku polecam Panel Pro dla Allegro Ads - znacznie ułatwia pracę nad dużymi ilościami grup reklam oraz ofert (disclosure: jestem autorem)

**Reply by Lukasz-Zurek** [reply]
@Zvapo Zdajemy sobie sprawę z dużej potrzeby udostępnienia API dla Allegro ADS. Dziękuję za kolejną sugestię.

**Comment by Hovise** [comment]
Zdecydowanie polecam włączenie API dla dziennych danych ADS. Pozwoliłoby mi to uzyskać dzienne statystyki wydatków i konwersji, umożliwiając terminowe dostosowywanie strategii, co stanowi istotne wsparcie dla sprzedaży. Kolejna kwestia: dane dotyczące wydatków ADS są potrzebne zespołowi do oceny efektywności pracowników.

**Reply by Toumash** [reply]
Dokładnie w ten sposób korzystamy z naszego własnego api do Allegro Ads zarówno na kontach sprzedawców jak i agencji reklamowych :)

**Reply by speedku** [reply]
@Toumash obchodzicie jakoś API, czy macie po prostu do niego dostęp?

**Comment by Lukasz90** [comment]
Witam, czy coś się zmieniło w sprawie API ADS? Po wątku widzę, że od lat jest potrzebne API ADS a oficjalnej wiadomości czy jest czy nie nie ma.

**Reply by kubaslawski** [reply]
Oficjalna wiadomość właśnie jest taka, że nie udostępnią: 
'Z uwagi na czynniki biznesowe, Allegro nie zdecydowało się dotychczas na publiczne udostępnienie API dla Allegro ADS.'

Z tego co rozumiem, ten feature jest zarezerwowany dla dużych agencji, więc prawdopodobnie tam są jakieś większe współprace B2B i nie opłaca im się tego udostępniać szerzej.

**Comment by cgchow94** [comment]
If I understood not. It's really sad as ads is a good revenue stream for marketplaces and having an API would help a lot

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Lukasz-Zurek** [reply]
Thank you for your suggestion. We are aware of this, and at the same time we do not rule out the implementation of these functionalities.

**Comment by alle916043712-oss** [comment]
Hello: When will this ads API be available? Many developers urgently need this interface.

**Reply by MaciejFrackowiak** [reply]
No decision has yet been made regarding making these resources available via a public API.
